APE_finance.rst

```diff
@@ -0,0 +1,143 @@
+
+Having a clear set of principles around how to use these funds will facilitate spending them fairly and effectively. Additionally, it will make it easier to solicit donations and other contributions. The guidelines here are meant for community members to consider when proposing to use Project funds, for the community to consider when voting on proposals, and for the Finance Committee to use to guide the funding process.
+
+A key assumption of this APE is that Astropy is developed and maintained by an institutionally and geographically distributed group of independently motivated individuals, and that the Project funds are not enough to employ most of the maintainers full-time. If funding increases to the point where that assumption is no longer true, this APE should be updated.
+
+In scope for this APE are:
+
+* Principles of spending project funds: what to pay for, and who to pay, and how much to pay.
+* Describing how Astropy’s funds get administered.
+* Describing how expenditure decisions are proposed and made.
+
+
+Principles for spending project funds
+-------------------------------------
+
+Astropy will likely always be a project with many times more volunteer contributors than funded people. It follows that having funded people operate in ways that attract more volunteers and enhance their participation experience is critical. That key principle motivates many of the more detailed principles given below for what to pay for and whom to pay.
+
+We identify four distinct considerations when disbursing funds:
+
+* Agreeing upon the types of activities that can be funded,
+* Establishing a process for deciding which specific activities should be funded,
+* Finding someone to do the work (if necessary),
+* Determining an appropriate compensation level.
+
+The following sections go into detail on each of these points.
+
+What to pay for
+^^^^^^^^^^^^^^^
+We can pay for anything which contributes to the broad mission of the Astropy Project. This includes, for example, code development, community engagement, education, promotion, etc. Funds can be used for many purposes, for example, paid developer time, teaching buyout, development of training materials, expenses for developer meetings or sprints, hosted hardware, and CI or other software services, carbon offsets, promotion and advertising, travel to conferences or meetings to promote Project activities.
+
+How to decide what to pay for
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+1. Pay for things that are important and otherwise won’t get done. *Rationale: there is much more work to be done than there are funds to do all of that work. There are also many volunteers and externally sponsored people interested and willing to do some of the work without compensation from the Project.*
```

Comment:
```suggestion
1. Pay for things that are important and otherwise will not get done. *Rationale: there is much more work to be done than there are funds to do all of that work. There are also many volunteers and externally sponsored people interested and willing to do some of the work without compensation from the Project.*
```